schemas/layer-2.cue

```diff
@@ -70,9 +70,17 @@ package schemas
     url?: =~"^https?://[^\\s]+$"
 }
 
+
 #Mapping: {
-    "reference-id": string @go(ReferenceId)
-    identifiers: [...string]
+	"reference-id": string @go(ReferenceId)
+	entries: [...#MappingEntry]
+	remarks?: string
+}
+
+#MappingEntry: {
```

Comment:
I feel like we want to avoid duplicating types across the schema and go packages.

Perhaps there is a way to define each layer schema as a collection of cue files where one or more of those files may contain types like those one which is reused across multiple layers